tests/integration-tc/src/main/java/org/apache/pulsar/tests/integration/FUNC_MODE.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.pulsar.tests.integration;
+
+public enum FUNC_MODE {
```

Comment:
I know this was originally for pulsar functions. but it might be good to rename it to a better name. 

probably `TEST_MODE`